aea/aea_builder.py

```diff
@@ -819,16 +816,37 @@ def build(
             decision_maker_handler_class=self._get_decision_maker_handler_class(),
             skill_exception_policy=self._get_skill_exception_policy(),
             default_routing=self._get_default_routing(),
+            default_connection=self._get_default_connection(),
             loop_mode=self._get_loop_mode(),
             runtime_mode=self._get_runtime_mode(),
             **deepcopy(self._context_namespace),
         )
         # load connection
-        self._load_and_add_connections(aea, wallet, connection_ids=connection_ids)
-        aea.multiplexer.default_routing = self._get_default_routing()
-        self._load_and_add_skills(aea.context, resources)
+        # TODO check address is the same of identity address
+        # TODO filter connections with connection_ids
```

Comment:
I think filtering is now done inside the AEA (call to `connect`) rather than outside...What am I missing?